### PR TITLE
Add redirect for falcon articles

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -15,4 +16,14 @@ app.get('/survey', function(req, res) {
 	}
 });
 
+// redirect falcon-style article routes
+app.get(/(\/intl)?\/cms\/s\/\d\/([a-f0-9\\-]+)\.*/, function(req, res){
+	var reqUrl = url.parse(req.url);
+	var uid = req.params[1];
+	var redirectUrl = '/' + uid + (reqUrl.search || '');
+	res.redirect(301, redirectUrl);
```
Comment (commuterjoy) [inline app.js]: Will sending out cache headers mean this can be cached in fastly?

File: test/app.spec.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+var request = require('supertest');
+var app = require('../app');
+
+describe('Redirector', function(){
+
+	describe('Falcon Article Urls', function(){
+
+		it('Should redirect conditional standard articles', function(done){
+			var url = "/cms/s/0/2fb5567e-b686-11e4-a5f2-00144feab7de.html";
```
Comment (commuterjoy) [inline test/app.spec.js]: Your tests don't test the _intl_ reg ex

File: app.js (hunk 1 above)
Comment (aintgoin2goa) [inline app.js]: I thought Varnish would cache a 301 automatically.  Turns out it doesn't, so I've added Cache-Control header

File: test/app.spec.js (hunk 2 above)
Comment (aintgoin2goa) [inline test/app.spec.js]: Yeah, took the regex from ft.com's access_metadata file (as useful list of regexes to match every page on ft.com) but could't find a real world example.  Have made one up.

Am passing the /intl/ bit on, is that right? Will Grumman know what to do with that?

Comment (commuterjoy) [inline test/app.spec.js]: Nope. We want to banish `intl` from the URL, so strip it out. We will vary internally on the x-ft-edition.

**Comment by aintgoin2goa** [comment]
Can I merge this now?

File: test/app.spec.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+			var expected = "/773961a2-ac59-11e4-9d32-00144feab7de";
+			request(app)
+				.get(url)
+				.expect('Location', expected)
+				.expect(301, done);
+		});
+
+		it('Should redirect articles including querystrings', function(done){
+			var url = "/cms/s/0/c0a015dc-b679-11e4-a5f2-00144feab7de.html?siteedition=uk";
+			var expected = "/c0a015dc-b679-11e4-a5f2-00144feab7de?siteedition=uk";
```
Comment (commuterjoy) [inline test/app.spec.js]: let's strip query strings too.

**Pull request merged**